### ISSUE TITLE
Don't duplicate starting salary data

### ISSUE THREAD

**Issue by aterrype** [bendyworks/payroll#206]
The starting_salary attribute on the employee model duplicates the salary information we already store in the salary table. We should remove the employee attribute so we aren't storing the same info twice.

**Comment by rogerroelofs** [comment]
This looks like a good starter for me.  @aterrype, what do you think?

**Comment by rogerroelofs** [comment]
First question:  If an employee is rehired, should the starting_salary be the original starting salary or the salary at rehire?

**Comment by aterrype** [comment]
Yeah, I think this would be a good starter. As for your question, I think the idea is that if we get rid of starting_salary it wouldn't matter - we'd just get the first salary in the salary db instead. (Which would be the original starting salary, but that's how it works now so I think that's fine.)

**Comment by rogerroelofs** [comment]
@aterrype  and @Code47X  Now that I've been poking around a bit I'm starting to wonder if salaries should be related to tenures rather than the employee.  Doing so would allow for having a starting and ending salary per tenure but feels like it is going to make getting data for charts more complicated.  Any thoughts?

**Comment by aterrype** [comment]
It would make sense for start/end dates, but I wonder how that would work for raises, since tenure doesn't care about raise dates. Another possibility might be to relate salaries and tenures through employees - I know you can do that in Rails, though I'm not super clear on the specifics so I can't say for sure that it would work here.

**Comment by Code47X** [comment]
I haven't looked too deeply into the data structure so far, but it makes sense to me to have salaries related to tenures. As @aterrype mentioned, maybe we just set it up as Employee `has_many :salaries, through: :tenures`.
